### PR TITLE
Change naming scheme in 32CSP to better reflect structure

### PR DESCRIPTION
**Pull request by H3w9VH3F4L4Tq5MbsfRi**

### CODE REVIEW

File: 3-coloring-algorithm/32CSP.cs [hunk 1]
```diff
@@ -150,13 +150,13 @@ public IEnumerable<((int index, ColorEnum color) v1, (int index, ColorEnum color
         {
             List<((int index, ColorEnum color) v1, (int index, ColorEnum color) v2)> toReturn = new();
 
-            var verticies = Vertices();
+            var verticies = Nodes();
```
Comment (Sosssen) [inline 3-coloring-algorithm/32CSP.cs:153]: verticies -> vertices -> nodes

**Review by Sosssen — CHANGES_REQUESTED** [review]

File: 3-coloring-algorithm/32CSP.cs [hunk 2]
```diff
@@ -56,12 +56,12 @@ public IEnumerable<int> Vertices()
...
 
-        public int VertexCount()
+        public int NodesCount()
         {
-            return Vertices().Count();
+            return Nodes().Count();
         }
 
-        public IEnumerable<ColorEnum> VertexColors(int vertex)
+        public IEnumerable<ColorEnum> NodeColors(int vertex)
```
[the remark this answers is not in the document]
Comment (H3w9VH3F4L4Tq5MbsfRi) [reply 3-coloring-algorithm/32CSP.cs]: done

File: 3-coloring-algorithm/32CSP.cs (hunk 1 above)
Comment (H3w9VH3F4L4Tq5MbsfRi) [reply 3-coloring-algorithm/32CSP.cs]: done

**Review by Sosssen — APPROVED** [review]
dej

**Pull request merged**